app/assets/stylesheets/shared.scss

```diff
@@ -61,3 +61,26 @@
     top: 0;
   }
 }
+// Decorate Choose file btn => "Choose Avatar"
+  #user_profile_image {color: transparent;}
+  #user_profile_image::-webkit-file-upload-button {visibility: hidden;}
+  #user_profile_image::before {
+  content            : 'Choose Avatar';
```

Comment:
Hey @dhoangk07 is there any way to get this from the locale files so it can be localized?

Reply:
Hey @scott 

I've already add I18n for it, so now it can be localized.

`data-content' => I18n.t('devise.registrations.choose_avatar')`